tests/test_types.py

```diff
@@ -955,6 +957,32 @@ class Model(BaseModel):
         Model(v=b'1')
 
 
+def test_strict_int():
+    class Model(BaseModel):
+        v: StrictInt
+
+    assert Model(v=123456).v == 123456
+
+    with pytest.raises(ValidationError):
```

Comment:
use `match=...`

Reply:
and also in all the other "raises" tests you've added below.

Reply:
Will do.